Added Organization name length limitation (#13147)
MicrosoftDocs/azure-devops-docs: docs/includes/organization-name-limitation.md

---

> [!IMPORTANT]

> > Currently, you can only use letters from the English alphabet in your organization name. Start organization names with a letter or number, followed by letters, numbers or hyphens, and they must end with a letter or number. And it must not contain more than 50 characters in length.

Reviewer: ```suggestion
> Currently, you can only use letters from the English alphabet in your organization name. Start organization names with a letter or number, followed by letters, numbers or hyphens. Organization names can't be longer than 50 characters and must end with a letter or number.
```
rewriting to avoid starting the sentence with 'and'
